app/src/main/java/com/example/mpweather/MainActivity.kt

```diff
@@ -93,8 +91,8 @@ class MainActivity : AppCompatActivity() {
                     tvResult!!.setTextColor(Color.rgb(68, 134, 199))
                     // Create the output string by concatenating the necessary variables properly
                     output += """Current weather of $cityName ($countryName)
- Temp: ${df.format(temp)}  °C
- Feels Like: ${df.format(feelsLike)}  °C
+ Temp: ${decimalFormatting.format(temp)}  °C
+ Feels Like: ${decimalFormatting.format(feelsLike)}  °C
```

Comment:
I replaced the previous name that was assigned to the variable to the new, more descriptive name.